File: Java/refactoring_exercise_final/src/main/java/za/co/entelect/refactoring_final/service/BankingService.java
package za.co.entelect.refactoring_final.service;

import za.co.entelect.refactoring_final.domain.BankAccount;

import java.util.ArrayList;
import java.util.List;


public class BankingService {

    private List<BankAccount> bankAccounts = new ArrayList<BankAccount>();

    public int countBanksAccounts() {
        return bankAccounts.size();
    }

    public void addBankAccount(BankAccount bankAccount){
        bankAccounts.add(bankAccount);
    }

}

File: C#/Refactoring/refactoring_exercise_final/za/co/entelect/refactoring_final/service/BankingService.cs
using System.Collections.Generic;
using refactoring_exercise_final.za.co.entelect.refactoring_final.domain;

namespace refactoring_exercise_final.za.co.entelect.refactoring_final.service
{

    public class BankingService
    {
        private readonly List<BankAccount> _bankAccounts = new List<BankAccount>();

        public int CountBanksAccounts()
        {
            return _bankAccounts.Count;
        }

        public void AddBankAccount(BankAccount bankAccount)
        {
            _bankAccounts.Add(bankAccount);
        }
    }
    
}